bin/install_geoserver.sh

```diff
@@ -31,10 +31,11 @@ USER_HOME="/home/$USER_NAME"
 TMP="/tmp/build_geoserver"
 INSTALL_FOLDER="/usr/local/lib"
 BIN="/usr/local/bin"
-GS_VERSION="2.10.4"
+GS_VERSION="2.13.1"
 GS_HOME="$INSTALL_FOLDER/geoserver-$GS_VERSION"
 GS_PORT=8082
 DOC_DIR="$GS_HOME/doc"
+GS_JAVA_HOME="/usr/lib/jvm/java-8-openjdk-amd64"
```

Comment:
This is going to be fixed in the Java installer, so that we have openjdk-8 as default